serverless/containers/concepts.mdx

```diff
@@ -40,7 +40,7 @@ An environment variable is a variable whose value is set outside the program, ty
 
 In addition to VCPU and RAM, Serverless Containers also provide a storage volume for the duration of the task. This storage space allows to hold the data retrieved by the job, and disappears once the execution is complete. The maximum size of the ephemeral storage is tied to the allocated memory.
 
-## GB-s
+## GB/s
```

Comment:
Hi @thomas-tacquet, we're not sure about this change, as GB-s is what we have in our [guidelines](https://www.scaleway.com/en/docs/guidelines/#bits-and-bytes), and seems to be in line with other cloud providers, when it comes to the unit to measure Serverless consumption.

GB/s = data transfer rate
GB-s = quantity of data stored or processed over a specific duration

What do you think, is there a specific reason you wanted to make this change?